trace/cloud-trace-demo-app/app/app.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A sample app demonstrating Stackdriver Trace
+"""
+
+import google.auth
+from flask import Flask, render_template, request
+from opencensus.trace import execution_context
+from opencensus.trace.propagation import google_cloud_format
+from opencensus.trace.samplers import AlwaysOnSampler
+from opencensus.ext.stackdriver.trace_exporter import StackdriverExporter
+from opencensus.ext.flask.flask_middleware import FlaskMiddleware
+
+import requests
+import argparse
+import time,random
+
+app = Flask(__name__)
+
+propagator = google_cloud_format.GoogleCloudFormatPropagator()
+
+def createMiddleWare(exporter, sampler):
+  # Configure Tracing
```

Comment:
This comment doesn't seem particularly useful. Can you expand more, maybe explain when this needs to be called in order to act properly as middleware?

Reply:
@dukeng Do you mind adding some comments for middleware? Thanks! :)

Reply:
Done